### DISCUSSION TITLE
Song Folder Settings

### DISCUSSION THREAD

**Discussion by AHungryPqqpy** [ppy/osu#22569] [Ideas]
Better Options other then importing songs, Being able to use the standard clients Beatmap's without it copying it to your OS file drive building up the storage space, sorry if my English is poor.

**Comment by bdach** [comment]
A second copy of all files will not be created if you use the "hard link" feature added recently. Please read https://osu.ppy.sh/wiki/en/Client/Release_stream/Lazer/File_storage#via-hard-links for an explanation. The first setup overlay will tell you as to whether hard links are used or not.

**Reply by AHungryPqqpy** [reply]
how would i set up hard links? is the only way to set them up is by reinstalling osulazer if i all ready did the setup?

**Reply by bdach** [reply]
As things stand, yes, if you've already got an installation and imported everything from stable you'd have to delete everything and import again to use hard links. This may change in the future.
